src/main/java/com/alkemy/ong/model/User.java

```diff
@@ -0,0 +1,55 @@
+package com.alkemy.ong.model;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.hibernate.annotations.Where;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+@SQLDelete(sql="UPDATE user SET deleted = true WHERE id=?")
+@Where(clause = "deleted=false")
+public class User {
+    
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "firstName", nullable = false, updatable = true)
```

Comment:
los "_name =_" son redundantes en este caso, porque le estas poniendo exactamente el mismo nombre que le definis en la clase, se usaría únicamente si el mismo cambia, para indicarle a la tabla de la DB que debe tener otro nombre y no el que se le puso en el codigo java.
Te falta también los mensajes de error para los atributos que no pueden ser nulos, por ejemplo @NotNull(message = "First name can't be null")

Reply:
Listo